<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimplePlayer</class>
 <widget class="QMainWindow" name="SimplePlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>416</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>523</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Demo Player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="VlcWidgetVideo" name="video" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>250</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="CurrentTimeLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="CustomSeekSlider" name="seekSlider">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="MediaDurationLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnPrevious">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="image.qrc">
          <normaloff>:/new/player of icon/image/左.ico</normaloff>:/new/player of icon/image/左.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stop">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="image.qrc">
          <normaloff>:/new/player of icon/image/停止.ico</normaloff>:/new/player of icon/image/停止.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pause">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="image.qrc">
          <normaloff>:/new/player of icon/image/暂停.ico</normaloff>:/new/player of icon/image/暂停.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnNext">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="image.qrc">
          <normaloff>:/new/player of icon/image/右.ico</normaloff>:/new/player of icon/image/右.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="fullButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="image.qrc">
          <normaloff>:/new/player of icon/image/全屏.ico</normaloff>:/new/player of icon/image/全屏.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="snapshotButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="image.qrc">
          <normaloff>:/new/player of icon/image/截图.ico</normaloff>:/new/player of icon/image/截图.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="modeButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="image.qrc">
          <normaloff>:/new/player of icon/image/顺序播放.ico</normaloff>:/new/player of icon/image/顺序播放.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="volumeButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="image.qrc">
          <normaloff>:/new/player of icon/image/音量.ico</normaloff>:/new/player of icon/image/音量.ico</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="CustomVolumeSlider" name="volumeSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="listButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="image.qrc">
          <normaloff>:/new/player of icon/image/列表.ico</normaloff>:/new/player of icon/image/列表.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>523</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpenLocal"/>
    <addaction name="actionOpenUrl"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuMedia">
    <property name="title">
     <string>媒体</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>播放模式</string>
     </property>
     <addaction name="action_mode_1"/>
    </widget>
    <addaction name="actionPause"/>
    <addaction name="action_mute"/>
    <addaction name="actionStop"/>
    <addaction name="actionfull"/>
    <addaction name="menu_2"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="action_list"/>
    <addaction name="actionsnapshut"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMedia"/>
   <addaction name="menu"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>暂停</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>停止</string>
   </property>
  </action>
  <action name="actionOpenLocal">
   <property name="text">
    <string>打开本地</string>
   </property>
  </action>
  <action name="actionOpenUrl">
   <property name="text">
    <string>打开网络流</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionsnapshut">
   <property name="text">
    <string>截图</string>
   </property>
  </action>
  <action name="actionfull">
   <property name="text">
    <string>全屏</string>
   </property>
  </action>
  <action name="action_mute">
   <property name="text">
    <string>静音</string>
   </property>
  </action>
  <action name="action_mode_1">
   <property name="text">
    <string>顺序播放</string>
   </property>
  </action>
  <action name="action_list">
   <property name="text">
    <string>打开播放列表</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VlcWidgetVideo</class>
   <extends>QWidget</extends>
   <header location="global">VLCQtWidgets/WidgetVideo.h</header>
   <container>1</container>
   <slots>
    <signal>fullscreen()</signal>
    <slot>toggleFullscreen()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>CustomSeekSlider</class>
   <extends>QSlider</extends>
   <header>customseekslider.h</header>
  </customwidget>
  <customwidget>
   <class>CustomVolumeSlider</class>
   <extends>QSlider</extends>
   <header>customvolumeslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>SimplePlayer</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
